### ISSUE TITLE
Accessibility - <html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by sarahyjja** [victormasson21/FACOnePageWebsite#1]
Please, have a look at the top of your project. In your html tag, add the attribute lang
-> lang="eng"
